src/modules/utils/panel/screens/FileScreen.cpp

```diff
@@ -185,3 +187,22 @@ void FileScreen::play(const char *path)
     message.stream = &(StreamOutput::NullStream);
     THEKERNEL->call_event(ON_CONSOLE_LINE_RECEIVED, &message );
 }
+
+void FileScreen::delete_file_sorter(void)
+{
+    if ( this->file_sorter ) {
```

Comment:
There is no need to check if `file_sorter` is non NULL, since C++ guarantees that `delete` of a NULL is ok.